client/src/main/java/com/kritsit/casetracker/client/domain/ui/controller/AdministratorController.java

```diff
@@ -122,6 +145,36 @@ public void initialize(){
         btnEdit.setOnAction(event->{
            editUser();
         });
+        
+        exportItem.setOnAction(event->{
+            exportService = new Export();
```

Comment:
Can this be refactored into its own method same as editUser(), addUser(), etc have been?